ginga/misc/plugins/TVMark.py

```diff
@@ -1,6 +1,5 @@
 """Non-interactive points marking local plugin for Ginga."""
-from __future__ import (absolute_import, division, print_function,
-                        unicode_literals)
+from __future__ import absolute_import, division, print_function
```

Comment:
:+1: From what documentation I can find, either we should import `unicode_literals` everywhere or not at all.  I know that for some 3rd party python2 libraries it causes problems.  I had to track down an obscure bug with another module where this had been inserted.

I think with an eventual move to python3 only it will not be necessary anyway so I'm in favor of not using it if we can avoid it.